### ISSUE TITLE
Add support for Goland

### ISSUE THREAD

**Issue by kkrav3ts** [JetBrains/intellij-micropython#325]
I am migrating from PyCharm to Goland and Micropython is the thing I am missing. Could you please advise whether it is possible in theory to add support for it in Goland and if so, I think it might be a good feature. Thanks!

**Comment by elmot** [comment]
Is there a point to have both mpy and go in the same project?

**Comment by kkrav3ts** [comment]
You are right, this is not a must-have. However, I see cases when I need audit mpy projects and working on another project in goland. 

For now I am using both Goland/Pycharm, but for the sake of minimizing number of windows and potentially licenses payments, I’d be glad to have that plugin in Goland.
